### PR TITLE
Add missing window keys (<C-w><C-[hjklovq]>)

### LINKED ISSUE #2582: <C-w><C-l> doesn't work (but <C-w>l works)
# Is this a BUG REPORT or FEATURE REQUEST?

Bug report

# What happened

Nothing happened.

The following keymappings does not work:

* `<C-w><C-l>`
* `<C-w><C-k>`
* `<C-w><C-j>`
* `<C-w><C-h>`

# What did you expect to happen

Same as `<C-w>l`, `<C-w>k`, `<C-w>j`, `<C-w>h`, jump to the specified window.

# How to reproduce it (as minimally and precisely as possible)

Always happened when the above 4 key sequences are typed

# Environment

- Extension (VsCodeVim) version: 71f41af8af0ecfb32d4bb980acb9493d50577dce
- VSCode version: 1.22.2
- OS version: Windows 10 Pro 64bit

### PR DESCRIPTION
**Pull request by tyru**
# What this PR does / why we need it

Some commands are even implemented but only keymappings are missing.
This PR makes it to work just to add the keymappings to invoke the implementation.

# Which issue(s) this PR fixes

Fixes #2582, #2467

### CODE REVIEW

**Comment by jpoon** [comment]
If you could do all the `ctrl` keys that would be most appreciated. However, I would also understand if it's too much work.

File: package.json [hunk 1]
```diff
@@ -330,11 +335,6 @@
         "command": "list.collapse",
         "when": "listFocus && !inputFocus"
       },
-      {
-        "key": "g g",
```
Comment (jpoon) [inline package.json]: We need this one. Mind putting it back?

Comment (tyru) [reply package.json]: After removing this, `gg` seems to work.

I'm new to VSCode extension development and I'm not used to package.json and what should I write to `contributes.commands`. so please correct me if I'm wrong.
But unlike other VSCode extensions, I guess it's enough to lists only one key sequences here to send to VSCodeVim's state machine.
The state machine processes keys one by one, and fire the command if key sequence is matched.

**Comment by tyru** [comment]
> If you could do all the ctrl keys that would be most appreciated.

Okay, I'll try it :)

**Comment by tyru** [comment]
Sorted `contributes.commands` alphabetically, but #2601 and another PR(s) have broken the master build.
Please wait merging this PR until the fix.

**Comment by tyru** [comment]
> #2601 and another PR(s) have broken the master build.

Sent #2609.
It fixes broken master.

But I found another problem.
The problem is that the test codes depend on user settings (settings.json).
After I changed [my config](https://gist.github.com/tyru/8f4d7bdfac08d3d83e859091ab99e93c) to `"editor.autoClosingBrackets": false`, the below test case failed.
because it assumes inputting `(` makes buffer content to `()`.

https://github.com/VSCodeVim/Vim/blob/16a234863b33635f688f8df796d7d1d6e31a1fd7/test/mode/modeInsert.test.ts#L209

Removing this line passes the test, however.

File: package.json [hunk 2]
```diff
@@ -72,209 +72,220 @@
...
-        "when": "editorTextFocus && vim.active && vim.use<D-v> && vim.mode == 'SearchInProgressMode' && !inDebugRepl"
+        "key": "ctrl+j",
+        "command": "extension.vim_navigateCtrlJ",
+        "when": "editorTextFocus && vim.active && vim.use<C-j> && !inDebugRepl"
       },
       {
-        "key": "ctrl+[",
-        "command": "extension.vim_ctrl+[",
-        "when": "editorTextFocus && vim.active && vim.use<C-[> && !inDebugRepl"
+        "key": "ctrl+k",
```
Comment (jpoon) [inline package.json]: I think we can merge the two `ctrl+k` to 

```
      {
        "key": "ctrl+k",
        "command": "extension.vim_ctrl+k",
        "when": "editorTextFocus && vim.active && vim.use<C-k> && !inDebugRepl"
      },
```

File: package.json [hunk 3]
```diff
@@ -72,209 +72,220 @@
...
-        "when": "editorTextFocus && vim.active && vim.use<D-c> && vim.overrideCopy && !inDebugRepl"
+        "key": "ctrl+u",
+        "command": "list.focusPageUp",
+        "when": "listFocus && !inputFocus"
       },
       {
-        "key": "ctrl+a",
-        "command": "extension.vim_ctrl+a",
-        "when": "editorTextFocus && vim.active && vim.use<C-a> && !inDebugRepl"
+        "key": "ctrl+v",
```
Comment (jpoon) [inline package.json]: I think we can merge these two `ctrl+v` as well:
```
      {
        "key": "ctrl+v",
        "command": "extension.vim_ctrl+v",
        "when": "editorTextFocus && vim.active && vim.use<C-v> && !inDebugRepl"
      },
```

File: package.json [hunk 4]
```diff
@@ -286,69 +297,58 @@
...
       {
-        "key": "h",
-        "command": "list.collapse",
-        "when": "listFocus && !inputFocus"
+        "key": "cmd+c",
+        "command": "extension.vim_cmd+c",
+        "when": "editorTextFocus && vim.active && vim.use<D-c> && vim.overrideCopy && !inDebugRepl"
       },
       {
-        "key": "g g",
```
Comment (jpoon) [inline package.json]: I'm fairly certain we need this. When the explorer window is selected, pressing `gg` will take you to the first item in the list.

**Comment by jpoon** [comment]
Thanks for sorting the `package.json`. Makes it easier to read/review.

**Comment by tyru** [comment]
> Test should pass on CI though? We need that test as we have has special logic to delete that matching ).
> 
> The tests already have a mechanism to override the "vim" specific configurations (https://github.com/VSCodeVim/Vim/blob/master/test/testConfiguration.ts), maybe we need to do something similar for the global settings.

Ah, okay, forgot about that 😅

File: package.json (hunk 4 above)
Comment (tyru) [reply package.json]: Oooops I'm sorry... entirely I didn't care the explorer window and the `when` condition`.
Will restore it later.

File: package.json (hunk 2 above)
Comment (tyru) [reply package.json]: Thanks. how about duplicate `ctrl+h` and `ctrl+j`?
In the same way, I should merge like this?

```
      {
        "key": "ctrl+h",
        "command": "extension.vim_ctrl+h",
        "when": "editorTextFocus && vim.active && vim.use<C-h> && !inDebugRepl"
      },
      {
        "key": "ctrl+j",
        "command": "extension.vim_ctrl+j",
        "when": "editorTextFocus && vim.active && vim.use<C-j> && !inDebugRepl"
      },
```

Comment (jpoon) [reply package.json]: Oh yeah, I didn't spot those duplicates. That looks good.

**Comment by tyru** [comment]
Okay, fixed problems 🎉 
Failing tests should pass if #2609 is merged.

**Comment by tyru** [comment]
Thanks @jpoon for merging, now this branch is all green.

All my branches are left except refactoring `p` implementation in visual-mode.
The refactoring said here https://github.com/VSCodeVim/Vim/pull/2609#issuecomment-386592394 is bit difficult, so please feel free to ship the next release :)

**Pull request merged**